tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1414,3 +1414,64 @@ describe('Test move cluster to newly created workspace and deploy application to
     })
   )
 });
+
+describe('Test Helm app with Custom Values', { tags: '@p1' }, () => {
+  const configMapName = "test-map"
+  const repoTestData: testData[] = [
+    {qase_id: 173, message: '`valuesFrom` with empty', path:'qa-test-apps/helm-app/values-from-with-empty-values' },
+    {qase_id: 174, message: '`valuesFrom` with NO', path:'qa-test-apps/helm-app/values-from-with-no-values' },
+    {qase_id: 175, message: '`valuesFiles` with empty', path: 'qa-test-apps/helm-app/values-files-with-empty-values' },
+    {qase_id: 176, message: '`valuesFiles` with NO', path: 'qa-test-apps/helm-app/values-files-with-no-values' }
+  ]
+
+  function createConfigMap() {
```

Comment:
This seems an useful and re-usable function. Why not make a `Cypress.Command`?

Reply:
I was giving a try to `function` hence used here, I will convert them to `Cypress.Command`. Thanks :)

Reply:
Updated.